app/main/model/image.py

```diff
@@ -30,8 +32,30 @@ class ImageModel(db.Model):
   __table_args__ = (
     db.Index('ix_images_context', context, postgresql_using='gin'),
   )
+
+  @property
+  def canned_response(self):
```

Comment:
Throw an exception if we get to the end of the function because `model` is not `pdq` nor `phash`? Alternatively, we can ripout `phash` at this point because it's not needed and just assume `IMAGE_MODEL` is always `pdq`